src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -124,6 +124,33 @@
  * }
  * </pre>
  * <p>
+ * Config for {@code eol} policy:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;LeftCurly&quot;&gt;
+ *   &lt;property name=&quot;option&quot; value=&quot;eol&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <pre>
+ * class Test {
+ *
+ *   void method() { /&#42; comment &#42;/ int a; // violation ''{' should have line break after'
```

Comment:
even example is best way to show how it works, please add it description section as text.
`Check see comment in code, but comments on a right from left curly are ignored.`

Reply:
done

Reply:
I think we can improve this a bit, let's do `Check is comment-aware, but ignores trailing comments following a left curly brace.`

Reply:
done